### PR TITLE
updated register modal

### LINKED ISSUE #70: Correct the login and Register error
Error occurs when the user has not entered all the fields.
Happens due to incorrect path for error message.

- [x] Fix it for login modal
- [ ] Fix it for register modal

### PR DESCRIPTION
**Pull request by ashwaniYDV**
Fixes #70 

Changes: [Add here what changes you made in this Pull Request and if possible provide links depicting the changes.]

Surge Deployment Link: 

Screenshots for the change: [Add here screenshots showing the changes.]

### CODE REVIEW

**Comment by ashwaniYDV** [comment]
@harsh-jindal What's field decorator?

File: src/components/RegisterModal/RegisterModal.js [hunk 1]
```diff
@@ -84,9 +85,22 @@ class RegisterModal extends Component {
...
+      }
+    });
+  };
+
   onChange = e => {
     this.setState({ [e.target.name]: e.target.value });
   };
+  onChangeSwitch = (checked,e) => {
+    let state={...this.state, isVendor: checked}
+    this.setState(state);
```
Comment (harsh-jindal) [inline src/components/RegisterModal/RegisterModal.js]: @ashwaniYDV `let state={...this.state, isVendor: checked}` is not required 
Just write `this.setState({isVendor:checked})`

**Review by harsh-jindal — CHANGES_REQUESTED** [review]

File: src/components/RegisterModal/RegisterModal.js (hunk 1 above)
Comment (ashwaniYDV) [reply src/components/RegisterModal/RegisterModal.js]: I've done it in that way first but it was showing errors

Comment (ashwaniYDV) [reply src/components/RegisterModal/RegisterModal.js]: because switch change event is on another function

Comment (harsh-jindal) [reply src/components/RegisterModal/RegisterModal.js]: Screenshot of error.
It doesn't matter on which function it is.

Comment (ashwaniYDV) [reply src/components/RegisterModal/RegisterModal.js]: sorry its ok now.
I'll update

Comment (ashwaniYDV) [reply src/components/RegisterModal/RegisterModal.js]: done

**Review by harsh-jindal — APPROVED** [review]

**Pull request merged**